CHANGELOG.md

```diff
@@ -72,15 +72,50 @@ Updated cycles wallet to a gzipped version of `20240410` release:
-- [129408](https://dashboard.internetcomputer.org/proposal/129408)
-- [129379](https://dashboard.internetcomputer.org/proposal/129379)
-- [129378](https://dashboard.internetcomputer.org/proposal/129378)
+- [129494](https://dashboard.internetcomputer.org/proposal/129494)
+- [129493](https://dashboard.internetcomputer.org/proposal/129493)
+- [129084](https://dashboard.internetcomputer.org/proposal/129084)
+- [129081](https://dashboard.internetcomputer.org/proposal/129081)
+- [129035](https://dashboard.internetcomputer.org/proposal/129035)
+- [128876](https://dashboard.internetcomputer.org/proposal/128876)
+- [128904](https://dashboard.internetcomputer.org/proposal/128904)
+- [128864](https://dashboard.internetcomputer.org/proposal/128864)
+- [128816](https://dashboard.internetcomputer.org/proposal/128816)
+- [128846](https://dashboard.internetcomputer.org/proposal/128846)
+- [128806](https://dashboard.internetcomputer.org/proposal/128806)
+- [128805](https://dashboard.internetcomputer.org/proposal/128805)
+- [128296](https://dashboard.internetcomputer.org/proposal/128296)
+- [128295](https://dashboard.internetcomputer.org/proposal/128295)
+- [128171](https://dashboard.internetcomputer.org/proposal/128171)
+- [128155](https://dashboard.internetcomputer.org/proposal/128155)
+- [128154](https://dashboard.internetcomputer.org/proposal/128154)
+- [128099](https://dashboard.internetcomputer.org/proposal/128099)
+- [128088](https://dashboard.internetcomputer.org/proposal/128088)
+- [127707](https://dashboard.internetcomputer.org/proposal/127707)
+- [127706](https://dashboard.internetcomputer.org/proposal/127706)
+- [127463](https://dashboard.internetcomputer.org/proposal/127463)
+- [127461](https://dashboard.internetcomputer.org/proposal/127461)
+- [127104](https://dashboard.internetcomputer.org/proposal/127104)
+- [127096](https://dashboard.internetcomputer.org/proposal/127096)
+- [127094](https://dashboard.internetcomputer.org/proposal/127094)
+- [127034](https://dashboard.internetcomputer.org/proposal/127034)
+- [127031](https://dashboard.internetcomputer.org/proposal/127031)
+- [126879](https://dashboard.internetcomputer.org/proposal/126879)
+- [126878](https://dashboard.internetcomputer.org/proposal/126878)
+- [126730](https://dashboard.internetcomputer.org/proposal/126730)
+- [126729](https://dashboard.internetcomputer.org/proposal/126729)
+- [126727](https://dashboard.internetcomputer.org/proposal/126727)
+- [126366](https://dashboard.internetcomputer.org/proposal/126366)
+- [126365](https://dashboard.internetcomputer.org/proposal/126365)
+- [126293](https://dashboard.internetcomputer.org/proposal/126293)
+- [126095](https://dashboard.internetcomputer.org/proposal/126095)
+- [126000](https://dashboard.internetcomputer.org/proposal/126000)
+- [125592](https://dashboard.internetcomputer.org/proposal/125592)
+- [125591](https://dashboard.internetcomputer.org/proposal/125591)
+- [125504](https://dashboard.internetcomputer.org/proposal/125504)
```

Comment:
A replica that incorporates these proposals was already included in 0.19.0.

The update-replica workflow used to filter this list, but something changed and it doesn't anymore.

```suggestion
```

Reply:
Comparing with https://dashboard.internetcomputer.org/governance?p=2&t=151&topic=TOPIC_REPLICA_VERSION_MANAGEMENT, the workflow generated list misses many proposals.

Reply:
I went through each one and checked for duplicates. it should only have unique proposals in the list now